### PR TITLE
Multiple source dirs

### LINKED ISSUE #55: Allow Horust to load services from multiple directories
When I run my services under Horust supervision, I often split them under different categories. For example, "Services that are core for my stack" or "Extra services that are optional".

These services often need to run in parallel. Both "Extra" ones and "Core", because one depends on another. I don't want to run ALL services by providing a root directory (for example, "Another Extra service pack" I don't want to run in one scenario).

For now, I was just using two different instances of Horust to do the job. One for core and another for dependent services.

But do I have to?

I could ( I guess, I have never tried to be honest ) use symlinks and link core services to dependent services.

Or I could extend Horust to accept more than one directory path, merge fetched services into one vector and pass it to validation, and then create one single instance of Horust to rule them all :)

Service `docker-compose` can read multiple `docker-compose.yml` by accepting multiple `-f docker-compose-1.yml -f docker-compose-2.yml` etc.

I think (thanks to StructOpt) it is trivial to implement such behavior in Horust. 
It wouldn't be a breaking change (Horust would still accept a single argument), and it would make my life much easier.

Example:
```
horust --service-path ./services/core --service-path ./services/extra
```

### PR DESCRIPTION
**Pull request by frondeus**
### Motivation and Context

Fixes #55 

### Description
* Change --service-path to accept multiple `PathBuf`
* Add `Horust::fetch_service_dirs` method
* Keep in memory list of directories, not an option

### How Has This Been Tested?

I tested it manually on Linux machine plus I created a test that checks if all services were loaded from two different locations.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (a change to man pages or other documentation)

### Checklist:

- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

**Comment by FedericoPonzi** [comment]
Awesome job, thanks!

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -191,10 +192,20 @@ command= "bash /tmp/myscript.sh"
 This will run the specified command as a one shot service, so it won't be restarted after exiting.
 Commands have precedence over services, so if you specify both a command and a services-path, the command will be executed and the services path is ignored.
 
+## Multiple service directories
+
+Horust supports more than one service directory by accepting more than one `--services-path` arguemts.
```
Comment (FedericoPonzi) [inline DOCUMENTATION.md:197]:
```suggestion
Horust supports more than one service directory by accepting more than one `--services-path` arguments.
```

**Review by FedericoPonzi — CHANGES_REQUESTED** [review]
Just one typo, otherwise LGTM

**Comment by esavier** [comment]
@FedericoPonzi can we expect cargo release somewhere soon?

**Comment by FedericoPonzi** [comment]
@esavier I've just pushed the latest cargo release (0.1.3) thanks!

**Pull request merged**